index.bs

```diff
@@ -3277,7 +3277,7 @@ the following algorithm returns <i>true</i>.
         <ul>
             <li>
                 <code>{{double}}?</code> and
-                <code>{{record}}&lt;{{DOMString}}, {{DOMString}}></code>
+                <code><a>record</a>&lt;{{DOMString}}, {{DOMString}}></code>
```

Comment:
Think you could use the [=record=] "shorthand" here for consistency with the rest of the spec?